go.sum

```diff
@@ -6,9 +6,12 @@ github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03
 github.com/Microsoft/go-winio v0.4.14 h1:+hMXMk01us9KgxGb7ftKQt2Xpf5hH/yky+TDA+qxleU=
```

Comment:
Looks like this might need a `go mod tidy`?

Reply:
👍  I've run it.